package.json

```diff
@@ -76,6 +76,7 @@
     "sinon": "^17.0.1",
     "sinon-chai": "^3.5.0",
     "tar-stream": "^3.1.7",
+    "undici": "^6.19.2",
```

Comment:
What do we need this dependency for? I don't see it used.

Reply:
My bad, I found it: https://github.com/camunda/camunda-modeler/pull/4409/commits/3880fab28c6efc574ad3a38bbd0922161e459885#diff-36079595e5a6a274b3943ab8ee0038f6faa22c5280098897fde1c59b2bad3673R16. :heavy_check_mark:

Reply:
The new way of mocking fetch since [Node 18](https://nodejs.org/en/blog/announcements/v18-release-announce#fetch-experimental).